provisio-maven-plugin/src/test/java/ca/vanzyl/maven/plugins/provisio/ValidatorIntegrationTest.java

```diff
@@ -50,7 +50,7 @@ public void testIncomplete()
 
         maven.forProject(basedir)
                 .execute("provisio:validateDependencies")
-                .assertLogText("[ERROR] Failed to execute goal ca.vanzyl.provisio.maven.plugins:provisio-maven-plugin:1.0.16-SNAPSHOT:validateDependencies (default-cli) on project basic: Missing dependencies: junit:junit:jar:4.13.2, org.hamcrest:hamcrest-core:jar:1.3 -> [Help 1]");
+                .assertLogText("[ERROR] Failed to execute goal ca.vanzyl.provisio.maven.plugins:provisio-maven-plugin:1.0.16-SNAPSHOT:validateDependencies (default-cli) on project basic: Missing dependencies: junit:junit:jar:4.13.2 -> [Help 1]");
```

Comment:
Remove the references to version numbers from the tests - otherwise they will need updating after every release.

Reply:
This was done on master. It's good now after rebasing.